### PR TITLE
fix: [IDP-3902]: Adding icon of Atlassian

### PR DESCRIPTION
**Pull request by agnivabanerjeeharness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2721

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2759

File: packages/icons/src/atlassian.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" width="800px" height="800px" viewBox="0 0 256 256" preserveAspectRatio="xMidYMid">
```
Comment (samriddhi-daga) [inline packages/icons/src/atlassian.svg:1]: Please remove the width and height

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2769

**Comment by samriddhi-daga** [comment]
Please rebase your branch with main

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2775

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2779

**Review by mallick370 — COMMENTED** [review]
lgtm

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2793

**Comment by samriddhi-daga** [comment]
Bump-up the version of package.json

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2795

File: packages/icons/src/atlassian.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 256 256" preserveAspectRatio="xMidYMid">
```
Comment (samriddhi-daga) [inline packages/icons/src/atlassian.svg:1]: Compress the svg

**Review by samriddhi-daga — APPROVED** [review]
LGTM

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2799

**Review by mallick370 — COMMENTED** [review]
LGTM

**Review by mallick370 — COMMENTED** [review]
lgtm

**Review by mallick370 — APPROVED** [review]

**Pull request merged**